packages/basti-cdk/src/basti-access-security-group.ts

```diff
@@ -0,0 +1,68 @@
+import { aws_ec2 } from 'aws-cdk-lib';
+
+import { generateShortId } from './basti-helper';
+import { TARGET_ACCESS_SECURITY_GROUP_NAME_PREFIX } from './basti-constants';
+
+import type { Construct } from 'constructs';
+import type { IBastiInstance } from './basti-instance';
+
+/**
+ * The properties for the bastion access security group.
+ */
+export interface BastiAccessSecurityGroupProps
```

Comment:
`BastiAccessSecurityGroupProps` should accept tags as well

Reply:
Hmm 🤔 Maybe we don't need to manage custom user-defined tags on our own and rely on the `Tags.of` method instead?

What I mean is that users will be able to add their tags this way:

```js
const bastiInstance = new BastiInstance(/*...*/);

Tags.of(bastiInstance).add('hello', 'world');
```

Looks like this is the unified approach in CDK. Let's see what other popular CDK libraries do

Reply:
This is fine from my point of view. 

Stack tags are propagated to all taggable resources

Reply:
Thank you for your input @DrFunk-n-stein!

Reply:
@DrFunk-n-stein, I would really appreciate it if you could share some overall feedback regarding the construct interface we came up with. I'm sure you have much more experience in CF/CDK than I do 😊

Reply:
```Tags.of(bastiInstance).add('hello', 'world');```

Should work like any other construct since we are just extending the `SecurityGroup`

Reply:
I'm a terraform guy 😅 started with CDK some weeks ago, so please forgive me that I can't give input right now. I think you got it covered

@BohdanPetryshyn